src/Powercord/plugins/pc-pluginManager/scss/_plugin.scss

```
@@ -77,30 +79,18 @@
     }
   }
 
+  &-permissions, &-footer {
+    border-top: 1px rgba(0,0,0,0.2) solid;
```

Maybe try using `--background-modifier-accent` here to match the color of Discord's settings dividers... :man_shrugging: 

```suggestion
    border-top: 1px var(--background-modifier-accent) solid;
```

---

True that looks much better, only later in the code found the background-modifiers myself, that was sadly before that point. Thanks for the suggestion :)